ipa-core/tests/compact_gate.rs

```diff
@@ -37,11 +37,13 @@ fn compact_gate_cap_8_no_window_semi_honest_plaintext_input() {
 }
 
 #[test]
+#[ignore]
```

Comment:
```suggestion
/// This test is turned off because of [`issue`]. 
///
/// This test will hang without `relaxed-dp` feature turned out until it is fixed
/// [`issue`]: https://github.com/private-attribution/ipa/issues/1298
#[ignore]
```